stm/stm.ml

```diff
@@ -1672,7 +1672,7 @@ end (* }}} *)
 
 and TacTask : sig
 
-  type output = Constr.constr * Evd.evar_universe_context
+  type output = (Constr.constr * Evd.evar_universe_context) option
```

Comment:
Would it make sense to name a type for just the constr, context pair, which is used below, and from that type derive the option type?